modules/snooker/CHANGELOG.md

```diff
@@ -8,13 +8,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 - `Warehouse` now contains the colgroup elements from the table.
+- `TableOperations` now includes functions to convert individual table cells to header cells (and vice-versa).
 
 ### Improved
 - Rows are now stored and tracked in the table model structures.
 
 ### Changed
 - `RowDetails` has been renamed to `RowDetail` and now takes a generic argument to define the type of cells stored.
 - `row` and `colgroup` generators are now passed the previous element during transform or modification operations.
+- Renamed `getColType` to `getColsType` in the `TableOperations` module to reflect it works with multiple columns.
```

Comment:
`supports` isn't the right word here and needs to be closer to `works`, so I'm not keen on that. Happy to other suggestions though.

Reply:
```suggestion
- Renamed `getColType` to `getColsType` in the `TableOperations` module to reflect it can lookup the type from multiple columns.
```
Maybe?